### ISSUE TITLE
It's hard to finetune the R2D-18

### ISSUE THREAD

**Issue by ydzheng1993** [facebookresearch/R2Plus1D#49]
I downloaded some models from Tutorial 5, and finetuned them on the UCF-101. Most of them work very well. But I found the pre-trained R2D-18 model is hard to finetune and improve the accuracy.  Is it possible that the link is wrong?

**Comment by liu-zhy** [comment]
I have the same problem.
